app/controllers/web-payments/apple-pay/merchantid-domain-association-controller.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+// NPM dependencies
+const logger = require('winston')
+
+// Local constants
+const { APPLE_PAY_MERCHANT_ID_DOMAIN_ASSOCIATION } = process.env
+
+module.exports = (req, res) => {
+  if (APPLE_PAY_MERCHANT_ID_DOMAIN_ASSOCIATION.length === 0) {
```

Comment:
If this environment variable doesn't exist for some reason this will throw an error that isn't handled (likely resulting in 500 stack trace sent to the client). 

Options: 
* Short circuit evaluation to make sure it exists as part of the evaluation
* Check to make sure environment is set up at the top and use that logic in the controller

Short circuit option: 
```suggestion
  if (APPLE_PAY_MERCHANT_ID_DOMAIN_ASSOCIATION && APPLE_PAY_MERCHANT_ID_DOMAIN_ASSOCIATION.length === 0) {
```

Reply:
> If this environment variable doesn't exist for some reason this will throw an error that isn't handled (likely resulting in 500 stack trace sent to the client).

I tested this and was surprised: I found there was no error and the `APPLE_PAY_MERCHANT_ID_DOMAIN_ASSOCIATION` constant was empty.

Reply:
Approved in any case–it does look less dangerous 💯